GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -214,6 +214,26 @@ public override string GVFSBinDirectoryName
 
             // Documented here (in the addressing section): https://www.unix.com/man-page/mojave/4/unix/
             public override int MaxPipePathLength => 104;
+
+            public override string UpgradeInstallAdvice
+            {
+                get { return "When ready, run " + this.UpgradeConfirmCommand + " to upgrade."; }
+            }
+
+            public override string UpgradeConfirmCommand
+            {
+                get { return "`sudo gvfs upgrade --confirm`"; }
+            }
+
+            public override string StartServiceCommand
+            {
+                get { return "`launchctl start org.vfsforgit.service`"; }

```

Comment:
@alameenshah / @jamill can you confirm this is the correct command.  I grabbed this from the start scripts.

Reply:
Spoke with @alameenshah on this and instead we should have "launchctl load /Library/LaunchAgents/org.vfsforgit.service.plist' .  We also tested it out and it worked as expected.  Updating now.